cmd/algocfg/profileCommand.go

```diff
@@ -19,38 +19,61 @@ package main
 import (
 	"bufio"
 	"fmt"
-	"github.com/spf13/cobra"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/spf13/cobra"
+
 	"github.com/algorand/go-algorand/cmd/util/datadir"
 	"github.com/algorand/go-algorand/config"
 	"github.com/algorand/go-algorand/util/codecs"
 )
 
 // profileConfigUpdater updates the provided config for non-defaults in a given profile
-type profileConfigUpdater func(cfg config.Local) config.Local
+type profileConfigUpdater struct {
+	updateFunc  func(cfg config.Local) config.Local
+	description string
+}
 
-// defaultConfigUpdater leaves all default values in place
-func defaultConfigUpdater(cfg config.Local) config.Local {
-	return cfg
+// validatorConfigUpdater leaves all default values in place
+var validatorConfigUpdater = profileConfigUpdater{
+	description: "Help ensure chain health by validating all blocks and transactions.",
+	updateFunc: func(cfg config.Local) config.Local {
+		cfg.CatchupBlockValidateMode = 0b1100
```

Comment:
what is 0b1100?

Reply:
This field is a bitmask, so I was trying to show which bits were set. It could also be set to 12 (which is what the documentation mentions).